CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 - Eliminate the parameter to specify a property in `globalCount()` and `viewportCount()`
+- Change type from `viewportFeatures` expression, adding vizProperties to features as in `Interactivity`
```

Comment:
I would put it in `Added` because as far as I know, this is not a breaking change, this just adds new functionality

Reply:
👍 Yeah, it is an addition, cause I'm finally keeping the "extended properties" for these features as they are, the ones required in the constructor (I had some doubts at the beginning...)